backend/sozisel/lib/sozisel_web/schema/types/quiz_types.ex

```diff
@@ -0,0 +1,43 @@
+defmodule SoziselWeb.Schema.Types.QuizTypes do
+  use SoziselWeb.Schema.Notation
+
+  object :quiz do
+    field :duration_time_sec, non_null(:integer)
+    field :target_percentage_of_participants, non_null(:integer)
+    field :tracking_mode, non_null(:boolean)
+    field :quiz_questions, non_null(list_of(:quiz_question))
```

Comment:
I created a convenience macro that replaces `non_null(list_of(non_null...` and I believe that it is in the Notation module. `strong_list_of(:quiz_question)`. Unless you alllow for nullable quiz questions of course.